app/views/admin/auctions/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>
+  <div class='usa-width-three-fourths'>Auctions</div>
+  <div class='usa-width-one-forth right'>
+    <a class='usa-button usa-button-green' href="/admin/auctions/new">Create a new auction</a>
+  </div>
+</h1>
+
+<div class='usa-grid-full'>
+<hr>
+<% @auctions.each do |auction| %>
+  <a href="/admin/auctions/<%= auction.id %>"><h3 class='usa-width-one-whole'><%= auction.title %></h3></a>
+  <div class='usa-width-one-whole'>
+    <div class='usa-width-one-sixth'>
+      <% if auction.available? %>
+        <%= image_tag "issue-icon", class: 'issue-icon' %>
+      <% else %>
+        <span class=''>not live</span>
+      <% end %>
+    </div>
+    <div class='usa-width-one-fourth'>
+      <b>Starts:</b> <%= Presenter::DcTime.convert(auction.start_datetime).to_s(:long) %>
```

Comment:
Small thing, but perhaps the DC-converted times should be methods on `Presenter::Bid`?

e.g.

```ruby
module Presenter
  class Auction
   # ...
   def start_dc_datetime
     Presenter::DcTime.convert(start_datetime).to_s(:long)
   end
   # ...
   end
end
```